app/models/gql/Objects.scala

```diff
@@ -726,6 +725,15 @@ object Objects extends Logging {
       )
     ),
     AddFields(
+      Field(
+        "approvedIndications",
+        OptionType(ListType(StringType)),
+        description = Some("Indications for which there is a phase IV clinical trial"),
+        resolve = c => {
+          val indications = DeferredValue(indicationFetcher.defer(c.value.id))
```

Comment:
@JarrodBaker Should it be coded explicitly The optional list returned with `deferOpt` instead `defer`?